pytorch/src/gen/java/org/bytedeco/pytorch/BlockArrayRef.java

```diff
@@ -41,8 +41,7 @@ public class BlockArrayRef extends Pointer {
 
   /** Construct an ArrayRef from a single element. */
   // TODO Make this explicit
-  public BlockArrayRef(@ByPtrRef Block OneElt) { super((Pointer)null); allocate(OneElt); }
-  private native void allocate(@ByPtrRef Block OneElt);
+  
```

Comment:
This seems like an odd thing to remove, but maybe not something that anyone actually uses? @sbrunk ?

Reply:
No. Things in `torch::jit` are for TorchScript support and we're not there yet. Besides, the constructor isn't available in the current bindings either.